ext/mbstring/tests/gh15824.phpt

```diff
@@ -0,0 +1,10 @@
+--TEST--
+GH-15824 (ValueError: mb_detect_encoding(): Argument #2 ($encodings) contains invalid encoding "UTF8")
+--EXTENSIONS--
+mbstring
+--FILE--
+<?php
+echo mb_detect_encoding('abc', 'UTF8, ASCII');
```

Comment:
Please add more test cases such that both mime names and aliases are tested.

Reply:
Added.